File: FizzBuzz.java
import java.util.Scanner;

class FizzBuzz
{
	public static void main(String[] args) {
		System.out.println("Enter the value you want to check up to.");

		Scanner input = new Scanner(System.in);
		int maxValue = input.nextInt();

		for (int i = 1; i <= maxValue; i++)
		{
			if (i % 3 == 0 && i % 5 == 0)
				System.out.print("Fizz Buzz!, ");
			else if (i % 3 == 0)
				System.out.print("Fizz!, ");
			else if (i % 5 == 0)
				System.out.print("Buzz!, ");
			else
				System.out.print(i + ", ");
		}
		
	}
}

File: FizzBuzz.cs
using System;

public class FizzBuzz
{
	public static void Main()
	{
		int maxNumber = 0;
		Console.WriteLine("Enter the amount of terms: ");
		maxNumber = Int32.Parse(Console.ReadLine());

		for (int i = 1; i <= maxNumber; i++)
		{
			if (i % 3 == 0 && i % 5 == 0)
				Console.Write("Fizz Buzz!, ");
			else if (i % 3 == 0)
				Console.Write("Fizz!, ");
			else if (i % 5 == 0)
				Console.Write("Buzz!, ");
			else
				Console.Write(i + ", ");
		}
		
  	}

}
